app/components/primer/flex_component.rb

```diff
@@ -37,30 +37,30 @@ class FlexComponent < Primer::Component
 
     # @example 100|Default
     #   <%= render(Primer::FlexComponent.new(bg: :gray)) do %>
-    #     <div class="p-5 border bg-gray-light">Item 1</div>
-    #     <div class="p-5 border bg-gray-light">Item 2</div>
-    #     <div class="p-5 border bg-gray-light">Item 3</div>
+    #     <%= render(Primer::BoxComponent.new(p: 5, classes: "border bg-gray-light")) { "Item 1" } %>
```

Comment:
Interesting. Are there not system arguments for the border/bg classes?

Reply:
Attempting to not get bogged down here...

```
<%= render(Primer::BoxComponent.new(p: 5, bg: :gray_light, border: true)) { "Item 1" } %>
```

which is what I assumed would work, did not:

![image](https://user-images.githubusercontent.com/2181356/107261497-9c4e8a00-69fc-11eb-910d-fddc6a7a88c1.png)

which we think is because of https://github.com/primer/view_components/blob/638d97c4a27d8fb3d993fdd617eda6cc704b2aa9/lib/primer/classify.rb#L216-L217

Reply:
Oh! Looks like we need to add support for `border: true` somehow. Mind filing a separate issue?